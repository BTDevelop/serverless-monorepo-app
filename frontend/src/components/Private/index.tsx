import * as React from 'react';
import { InjectedAuthReduxProps } from 'redux-auth-wrapper/history4/redirect';
import { Segment } from 'semantic-ui-react';

export class Private extends React.Component<InjectedAuthReduxProps, any> {
  public render() {
    return <Segment> This is a private page </Segment>;
  }
}

export default Private;
