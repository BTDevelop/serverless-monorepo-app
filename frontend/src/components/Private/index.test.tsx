import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Private from './';

describe('Private Component', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(
        <Private
          redirectPath=""
          isAuthenticated={true}
          isAuthenticating={false}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
