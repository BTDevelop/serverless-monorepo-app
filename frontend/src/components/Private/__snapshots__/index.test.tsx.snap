// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private Component should render correctly 1`] = `
<div
  className="ui segment"
>
  This is a private page
</div>
`;
